src/function/geometry/distance.js

```diff
@@ -233,6 +225,21 @@ export const createDistance = /* #__PURE__ */ factory(name, dependencies, ({ typ
     return _isNumber(a[0]) && _isNumber(a[1]) && _isNumber(a[2])
   }
 
+  function _isNdim (a) {
+    // checks if the number of arguments are correct in count and are valid (should be numbers)
+    if (a.constructor !== Array) {
```

Comment:
Can you use `!Array.isArray(a)` here instead of `a.constructor !== Array`?

Reply:
I had it this way to be consistent with _3d and _2d I didnt donsider what was best practice or fastest. I'll change it but leave the others just in case

Reply:
Ah, I see, this was already there. I think we should replace all occurrences with `!Array.isArray(a)` since relying on things like `.constructor` can be tricky in general.